contrib/local.mk.example

```diff
@@ -9,7 +9,7 @@
 
 # By default, the jemalloc family of memory allocation functions are used.
 # Uncomment the following to instead use libc memory allocation functions:
-# CMAKE_EXTRA_FLAGS="-DUSE_JEMALLOC=OFF"
+# DEPS_CMAKE_FLAGS := -DUSE_BUNDLED_JEMALLOC=OFF
```

Comment:
> I see, although I'm still unsure why there's no option to "hard" disable jemalloc (like USE_JEMALLOC=OFF). That way, jemalloc wouldn't be built and you wouldn't need to specify USE_BUNDLED_JEMALLOC=OFF as well as USE_JEMALLOC=OFF.

It would be wonderful if we could dependencies and Neovim built as required under one invocation of CMake.  Unfortunately, that's not very possible--at least not without a great deal of work and fragility.  If we attempted to do such a thing, then we could not correctly obtain flags required for the dependencies via pkg-config (because the would not be built at CMake's configure time, and thus would be unavailable for consumption).  Instead, we'd have to re-implement whatever logic each of them has into our build system, which is almost certain to be wrong in some way.  That's why there is this split-brained situation, and why there are two separate knobs.  We could do more in the Makefile to help, but I'd really like to keep that as slim as possible--we don't want two build systems.  CMake is *the* build system, Make is just some nice glue to make it easier to get up and running.  Truthfully, I don't use Make much with Neovim--I drive CMake directly.

> USE_JEMALLOC=OFF

You still have to pass it to both the deps and Neovim builds though (the flags aren't shared between both).  We could come up with some variable that we could share between both... I'm not sure if that's good or bad though.  I'd expect just as many issues when they start trying to pass flags that only makes sense for one and not the other.

> Anyways, it's a rather small inconsistency so perhaps it could be dealt with later.

That depends on how you look at it. :-)  It's certainly consistent with how the other dependencies are built and included, and they communicate different things.  There's one small section of overlap: when USE_JMALLOC is OFF.  But it's hardly right to say they completely overlap. :-)